.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1267,6 +1293,7 @@ public class ChatMessage : IJsonModel<ChatMessage>, IPersistableModel<ChatMessag
         public static AssistantChatMessage CreateAssistantMessage(ChatCompletion chatCompletion);
         public static AssistantChatMessage CreateAssistantMessage(ChatFunctionCall functionCall);
         public static AssistantChatMessage CreateAssistantMessage(params ChatMessageContentPart[] contentParts);
+        public static AssistantChatMessage CreateAssistantMessage(ChatOutputAudioReference audioReference);
```

Comment:
We should call the parameter "outputAudioReference" to match the name of the parameter in the AssistantChatMessage constructor.